connect/cli/plugins/project/renderer.py

```diff
@@ -0,0 +1,130 @@
+        if exclude and not isinstance(exclude, list):
+            raise TypeError('The parameter exclude is invalid, it must be a list.')
+
+    def _create_directories(self, output_dir):
+        excludes = [os.path.join(output_dir, pattern) for pattern in self.exclude]
+        for element in Path(self.template_folder).rglob('*'):
+            directory = os.path.join(output_dir, os.path.relpath(str(element), self.template_folder))
+            directory = Template(directory).safe_substitute(self.context)
+            if element.is_dir() and not fnmatch.filter(excludes, directory):
+                os.makedirs(directory, exist_ok=True)
+                console.print(
+                    f'Folder {directory.replace(output_dir, "")}'
+                    ' created [bold green]\u2713[/bold green]',
+                )
+
+    def render(self):
+        if self.overwrite:
+            purge_dir(self.output_dir)
+            console.print(f'Directory {self.output_dir} deleted.')
+
+        with tempfile.TemporaryDirectory() as tmpdir:
+            self._create_directories(output_dir=tmpdir)
+            if self.pre_render_function:
+                self.pre_render_function(tmpdir, self.context)
+                console.print('pre_render_function executed [bold green]\u2713[/bold green]')
+            with console.status('[magenta]Generating files'):
+                templates = self.env.list_templates()
+                for template in templates:
+                    self._render_template(template, tmpdir)
+                if self.post_render_function:
+                    self.post_render_function(tmpdir, self.context)
+                    console.print('post_render_function executed [bold green]\u2713[/bold green]')
+                for folder in os.listdir(tmpdir):
+                    shutil.move(
+                        os.path.join(
+                            tmpdir,
+                            folder,
+                        ),
+                        self.output_dir,
+                    )
+                console.print('moved generated tree to destination [bold green]\u2713[/bold green]')
+                console.print()
+
+    def _render_template(self, template_name, destination):
+        evaluated_template_path = Template(
+            str(template_name),
+        ).safe_substitute(
+            self.context,
+        )
+        if not fnmatch.filter(self.exclude, evaluated_template_path):
+            file_destination = os.path.join(
+                destination,
+                evaluated_template_path[:-3],
+            )
+            template = self.env.get_template(template_name)
+            content = template.render(self.context)
+            with open(file_destination, 'w') as outstream:
+                outstream.write(f'{content.rstrip()}\n')
+            console.print(
+                f'File {file_destination.replace(destination, self.output_dir)}'
```

Comment:
Please change this replace to: `file_destination.replace(destination, '')` like for folders.